wifiphisher.py

```diff
@@ -875,10 +877,37 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # get template_database
+    template_database = phishingpage.get_template_database()
+
+    # check to see if the template is local
+    if (phishingpage.exists(TEMPLATE_NAME) and
+            (not template_database[TEMPLATE_NAME] is None)):
+        # if template is incomplete locally, delete and ask for a download
+        if not phishingpage.check_template(TEMPLATE_NAME):
```

Comment:
Now the downloading only happens if the template is incomplete locally. If the user runs the tool for the first time and uses `linksys` as `TEMPLATE_NAME` he won't be asked to download it.